tests/contrib/tornado/test_executor_decorator.py

```diff
@@ -25,11 +25,13 @@ def test_on_executor_handler(self):
         assert 200 == response.code
 
         traces = self.pop_traces()
-        assert 1 == len(traces)
-        assert 2 == len(traces[0])
+        assert len(traces) in (1, 2)
```

Comment:
Hmmm I am not sure if this is the right fix. The tornado trace is now being split into two traces:
```
[
[<Span(id=4048838241015833674,trace_id=4485748532218864876,parent_id=None,name=tornado.executor.with)>], 
[<Span(id=5343199955087320001,trace_id=2529932930374973753,parent_id=None,name=tornado.request)>]
]
```

The toronado `tornado.executor.with` span should always be in the same trace as the `tornado.request` span. I am not sure how or why this changed. Did we make any recent changes to context propagation or the dummy tracer?